src/api/rosetta-constants.ts

```diff
@@ -40,7 +39,11 @@ export const RosettaOperationTypes = [
   'mint',
   'burn',
   'miner_reward',
-  'stx_lock',
+  'stx_lock', // stx event
+  'stx_unlock', // forged event
+  'stack_stx', // PoX contract function
+  'delegate_stx', // PoX contract function
+  'revoke_delegate_stx', // PoX contract function
```

Comment:
I don't see where some of these constants are used anywhere. For example I see `delegate-stx` used as the rosetta operation type here:
https://github.com/blockstack/stacks-blockchain-api/blob/d2af183033873e3957ed8277485465b00651af3f/src/rosetta-helpers.ts#L550
but not `delegate_stx` anywhere. 

Similar for `stack_stx  and `revoke_delegate_stx`.

Seems like these should all be defined as a enum or string literal type union, then used in all the areas where Rosetta operation types are checked or set.

Reply:
Looks like this was due to bad refactors which caused a mismatch between the operation types reported here and type actually being used. Should be fixed now in https://github.com/blockstack/stacks-blockchain-api/pull/689/commits/0d54de8b9daca11efbadb80b3df601db99029f5e